src/include/miopen/getitem/problem_description.hpp

```diff
@@ -114,6 +114,38 @@ struct ProblemDescription : ProblemDescriptionBase
         return true;
     }
 
+    bool IsValidIndexs() const
+    {
+        if(indexCount > 0)
+        {
+            if(indexDescs == nullptr)
+                MIOPEN_THROW(miopenStatusBadParm,
```

Comment:
(here and below) I recommend putting validity checks into ctor of ProblemDescription.

Reply:
...so ctor fails to construct wrong object.

Reply:
fixed it.